src/main/java/core/basesyntax/report/convertdata/DataConvertorImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.report.convertdata;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConvertorImpl implements DataConvertor {
+    private static final String CSV_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int REPORT_START_POSITION = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitOperation> convertToTransaction(List<String> fruitInfoList) {
```

Comment:
Follow SOLID rules: The method name 'convertToTransaction' could be improved to reflect its purpose more clearly. Consider renaming it to something like 'convertToOperations' or 'parseOperations', as it converts string data to a list of FruitOperation objects, not transactions.